pkgs/development/r-modules/wrapper.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, R, makeWrapper, recommendedPackages, packages }:
 
-stdenv.mkDerivation {
-  name = R.name + "-wrapper";
+stdenv.lib.appendToName "wrapper" (stdenv.mkDerivation {
```

Comment:
Why is this version better than what we have now? They seem to give the same result.

Reply:
I agree, I like the old version more given that it avoids one level of parenthesis nesting.

Reply:
The difference is where the version goes. AppendToName will put this before the version.

Reply:
Ah, you are right. That is the difference, indeed. :-)

As it happens, I actually want that change. I want the package name to be `R` and the version to be `3.5.0-wrapper`. I don't want the package to be called `R-wrapper`.

Reply:
Ah, you are right. That is the difference, indeed. :-)

As it happens, I actually don't want that change. I want the package name to be `R` and the version to be `3.5.0-wrapper`. I don't want the package to be called `R-wrapper`.

Reply:
Ok. Any opinions on the other 3? My assumption was that they were all mistakes, not realizing that the version is at the end of name.

Reply:
Thanks! I have no opinion on those other 3 cases, I don't maintain those packages.